src/editor/src/bramble/client/main.js

```diff
@@ -535,7 +535,7 @@ define([
                 setReadyState(Bramble.MOUNTING);
 
                 root = Path.normalize(root);
-                filename = Path.resolve(root, filename || (_state.fullPath || "index.html"));
+                filename = Path.resolve(root, filename || (_state.fullPath || "editor.html"));
```

Comment:
This change is wrong.  Here we're referring to the `index.html` page in the editor, not for the editor.